Content.Server/Administration/Commands/AGhost.cs

```diff
@@ -64,6 +67,10 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
                 mind.TransferTo(ghost);
             }
 
+            // So aghosts can see other aghosts hidden by the hide aghost command
+            if (_entMan.TryGetComponent(ghost, out EyeComponent? eyeComponent))
+                eyeComponent.VisibilityMask = (uint) VisibilityFlags.Ghost | (uint) VisibilityFlags.Aghost + 1;
```

Comment:
Is this ever cleaned up? If someone goes back to a regular ghost it'll still have aghost, also you should use the enum for Normal rather than 1.